dispatch/apps/events/models.py

```diff
@@ -47,6 +47,8 @@ class Event(Model):
 
     is_published = BooleanField(default=False)
 
+    slug = SlugField(max_length=255)
```

Comment:
You get something like `2017-06-20 12:19:13.111972` - the template language seems to format it differently